src/commands/client/types.ts

```diff
@@ -1,10 +1,13 @@
 export type Vector<TMetadata = Dict> = {
   id: string;
   vector?: number[];
+  sparseVector?: RawSparseVector;
   metadata?: TMetadata;
   data?: string;
 };
 
 export type NAMESPACE = string;
 
 export type Dict = Record<string, unknown>;
+
+export type RawSparseVector = [index: number[], factors: number[]];
```

Comment:
- Why do we call this raw sparse vector, instead of just sparse vector?
- What does the names `index` and `factors` mean?
- In pinecone and qdrant js clients, this is defined in a similar fashion to python client, where sparse vector is an object with two fields, indices and values. I think we need to do the same to be consistent with others and our python sdk. 

https://docs.pinecone.io/guides/data/query-sparse-dense-vectors#query-records-with-sparse-dense-values

https://qdrant.tech/documentation/concepts/vectors/#sparse-vectors

Reply:
I agree, I will name it like this:
```ts
export type SparseVector = [indices: number[], values: number[]];
```

The reason behind RawSparseVector was I originally planned to do it like this:
```ts
export type SparseVector = {indices: number[], values: number[]}
export type RawSparseVector = [indices: number[], values: number[]];
```

This way, users would be able to access the `indices` and `values` by name, like `sparseVector.indices`. Decided not to to keep it simple.